controllers/SyopsController.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const BaseController = require('./BaseController');
+const OrganisationSearchService = require("../services/UserManagement/OrganisationSearchService");
+const UserActionsService = require("../services/UserManagement/UserActionsService");
+const syopsDateCheck = require("../helpers/syopsDateCheck");
+const requestOptions = require('../helpers/requestOptions');
+const { iamApi } = require("../config");
+const logger = require('hmpo-logger').get();
+
+class SyopsController extends BaseController {
+  // re-running of middleware function is if page is navigated directly to, to not render accept button if accepted already
+  async getValues(req, res, next) {
+    try {
+      const data = await OrganisationSearchService.orgLookup({
```

Comment:
The name of this service makes less sense now we've expanded it's use. I'm not going to request this to be refactored (as you've already done loads here already), but can you add a quick todo comment saying that this would benefit from being renamed so we can ~ignore~ fix it in the future?!

Reply:
I've just done the refactor or it'll never get done...damn having experience!